Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -10,9 +10,11 @@
 from datetime import datetime
 import json
 import requests
+import warnings
```

Comment:
Is this part of default python? Or you needed to install it?

Reply:
didn't need to install it.

Reply:
Sounds good!